test/e2e/framework.go

```diff
@@ -37,6 +37,7 @@ import (
 
 const (
 	maxKubectlExecRetries = 5
+	KubeE2ETestTag        = "k8s.io-e2e"
```

Comment:
I don't think we need the suffix e2e, if anything it may be called "k8s.io/kubernetes" as one could extend the e2es to contrib or other hosted at the same location.